components/script/dom/htmliframeelement.rs

```diff
@@ -491,16 +492,8 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
     }
 
     // https://html.spec.whatwg.org/multipage/#attr-iframe-allowfullscreen
```

Comment:
link to WHATWG may break in the future, use this format instead: https://html.spec.whatwg.org/multipage/#dom-iframe-allowfullscreen